packages/sanity/src/core/releases/tool/detail/__tests__/ReleaseDetail.test.tsx

```diff
@@ -256,15 +262,19 @@ describe('after bundles have loaded', () => {
         fireEvent.click(screen.getByText('Publish all'))
         fireEvent.click(screen.getByText('Publish'))
 
-        expect(useBundleOperations().publishBundle).toHaveBeenCalledWith('test-id', [
-          {
-            _createdAt: currentDate,
-            _id: 'test-id',
-            _rev: 'abc',
-            _type: 'document',
-            _updatedAt: currentDate,
-          },
-        ])
+        expect(useBundleOperations().publishBundle).toHaveBeenCalledWith(
+          'test-id',
+          [
+            {
+              _createdAt: currentDate,
+              _id: 'test-id',
+              _rev: 'abc',
+              _type: 'document',
+              _updatedAt: currentDate,
+            },
+          ],
+          {'123': 'mock revision id'},
```

Comment:
NIT, but would it make sense to have the ids of this tests to be closer to what we are gonna be receiving?

e.g. 

```
.toHaveBeenCalledWith(
          'summer-drop',
          [
            {
              _createdAt: currentDate,
              _id: 'summer-drop.123',
              _rev: 'abc',
              _type: 'document',
              _updatedAt: currentDate,
            },
          ],
          {'123': 'mock revision id'},
```